sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -490,14 +490,24 @@ let { a, b, c, ...everythingElse }: MyProps = $props();
 >
 > ...TypeScript [widens the type](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwBIAHGHIgZwB4AVeAXnilQE8A+ACgEoAueagbgBQgiCAzwA3vAAe9eABYATPAC+c4qQqUp03uQwwsqAOaqOnIfCsB6a-AB6AfiA) of `x` to be `string | number`, instead of erroring.
 
-Props cannot be mutated, unless the parent component uses `bind:`. During development, attempts to mutate props will result in an error.
+By default props are treated as readonly, meaning reassignments will not propagate upwards and mutations will result in a warning. You will also get an error trying to `bind:` to them. To declare props as bindable, use [`$bindable()`](#bindable).
 
 ### What this replaces
 
 `$props` replaces the `export let` and `export { x as y }` syntax for declaring props. It also replaces `$$props` and `$$restProps`, and the little-known `interface $$Props {...}` construct.
 
 Note that you can still use `export const` and `export function` to expose things to users of your component (if they're using `bind:this`, for example).
 
+### `$bindable()`
+
+To declare props as bindable, use `$bindable()`. Besides using them as regular props, the parent can then also `bind:` to them.
+
+```svelte
+<script>
+	let { bindableProp = $bindable() } = $props();
+</script>
+```
+
```

Comment:
We should include an example of using a default prop value with `$bindable(...)`.

Reply:
Is a default value ever useful in `$bindable`? If one is given, then it can't be used, since the parent has to pass a non-undefined value, right? So the child shouldn't set a default and check for undefined instead, then set the default. At least I think that's how it works now.

Reply:
If the parent does not bind to the component but passes it as a regular prop, the fallback value will be respected.